.github/workflows/semantic-release.yml

```
@@ -57,19 +87,40 @@
       # Cache Jest
       - name: Cache Jest
         uses: actions/cache@v4
+        if: steps.check_cache_skip.outputs.skip_cache_detected != 'true'
         with:
           path: |
             node_modules/.cache/jest
           key: ${{ runner.os }}-jest-${{ hashFiles('**/jest.config.js') }}-${{ hashFiles('**/*.spec.ts') }}
           restore-keys: |
             ${{ runner.os }}-jest-
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)